### DISCUSSION TITLE
vscode wallbash

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#1004] [General]
Is it possible to enable wallbash in vscode

**Comment by mahaveergurjar** [comment]
Is it possible to enable wallbash in vscode 
@kRHYME7

**Reply by kRHYME7** [reply]
https://stackoverflow.com/questions/75859763/how-can-i-add-custom-css-styles-to-vs-code


Hmm maybe there's an extension to import css color. If you can help me find one it would be great

**Comment by mahaveergurjar** [comment]
Ok I research more on this i find but at this time my exam are comming so I will find after exam

**Comment by mahaveergurjar** [comment]
I can change the theme color like this so It is possible to add this in wallbash 
```css
//Theme customisation
  "workbench.colorCustomizations": {
    "[Tokyo Night Storm]": {
      "editor.background": "#352663",
      "terminal.background": "#2b2149",
      "activityBar.background": "#392d5c",
      "statusBar.background": "#352a57",
      "editorGroupHeader.tabsBackground": "#2e234e",
      "tab.inactiveBackground": "#322755"
    }
  },
``` 
![240401_20h30m55s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/f489e262-afe6-4131-920b-c06c0b3fc2a2)

**Reply by kRHYME7** [reply]
yup this is planned

**Reply by RookiexCookie** [reply]
you guys are absolute mad lads ♥️

**Comment by mahaveergurjar** [comment]
ok

**Reply by kRHYME7** [reply]
https://github.com/prasanthrangan/hyprdots/discussions/1235 Please spread.  This is just a soft launch

**Reply by mahaveergurjar** [reply]
Okay 👍

**Comment by ghost** [comment]
Now it is! https://github.com/prasanthrangan/hyprdots/commit/b9021ca85f45c672e6e65d8471bbb694a1eb424d
